src/deepsparse/server/build_logger.py

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Specifies the mapping from the ServerConfig to the DeepSparse Logger
+"""
+
+from typing import Any, Dict, Optional, Union
+
+from deepsparse import loggers as logger_objects
+from deepsparse.loggers.config import (
+    MultiplePipelinesLoggingConfig,
+    PipelineLoggingConfig,
+)
+from deepsparse.server.config import ServerConfig
+
+
+__all__ = ["build_logger"]
+
+SUPPORTED_LOGGER_NAMES = ["python"]
+
+
+def build_logger(server_config: ServerConfig) -> logger_objects.BaseLogger:
```

Comment:
I think this looks good so far, we'll probably need to construct a different logger per endpoint, which is different than the original design layed out. that'll just involve breaking apart this function you have here, so pretty easy